editor/src/clj/editor/tile_source.clj

```diff
@@ -293,7 +260,7 @@
                             :tile-source-attributes tile-source-attributes
                             :anim-data   (get anim-data id)
                             :start-tile  start-tile}
-                :passes    [pass/outline pass/overlay pass/selection]}
```

Comment:
removing of the selection pass in tile source broke animation play on `spacebar`:
https://forum.defold.com/t/defold-1-12-0-beta/81981/63?u=agulev